src/com/sun/ts/tests/jpa/core/entityManager/Client.java

```diff
@@ -276,6 +277,38 @@ public boolean verifyListEmployees(List<Employee> expected,
     return result;
   }
 
+  /*
+   * @testName: autoCloseableTest
+   *
+   * @assertion_ids: PERSISTENCE:JAVADOC:36
```

Comment:
@Tomas-Kraus do you agree with my suggestion in https://github.com/eclipse-ee4j/jakartaee-tck/pull/759#issuecomment-966435181 that we should use a higher number than `36`? 

I think that we should add to the highest ID used currently, so perhaps something like `PERSISTENCE:JAVADOC:3491`.  

Thoughts?

Reply:
Will use the current highest number `3491` and increment.  I'll make the change.

Reply:
I used spec reference instead. API lists are being generated directly from API build using script so manual changes are not a good idea here.